assets/src/selected-stories-block/block/edit.css

```diff
@@ -0,0 +1,254 @@
+/* Content Overlay Styles */
+.wp-block-web-stories-selected-stories .story-content-overlay {
+  position: absolute;
+  z-index: 1;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background-color: rgba(0, 0, 0, 0.6);
+  box-shadow: 0 0 12px rgba(0, 0, 0, 0.2);
+  padding: 20px;
+  color: #ffffff;
+  font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto,
+    Oxygen-Sans, Ubuntu, Cantarell, 'Helvetica Neue', sans-serif;
+  font-size: 12px;
+  line-height: 16px;
+  letter-spacing: 0.2px;
+}
+
+.story-content-overlay .story-content-overlay__author-date {
+  display: flex;
+  justify-content: space-between;
+  flex-wrap: wrap;
+  margin-top: 8px;
+}
+
+.story-content-overlay .story-content-overlay__title {
+  grid-column: 1/-1;
+  font-size: 15px;
+  line-height: 22px;
+  font-weight: 700;
+  display: box;
+  -webkit-line-clamp: 2; /* Browser support: https://caniuse.com/?search=line-clamp */
+  box-orient: vertical; /* stylelint-disable-line property-no-unknown */
+  overflow: hidden;
+}
+
+.wp-block-web-stories-selected-stories.is-view-type-list
```

Comment:
Can we not use the `latest-web-stories` style here, we may not need this entire file. Maybe add `selected-stories` as a separate class for identification.

Reply:
If we can use less/scss, then it'll be very easy to re use the style sheet. But I'm not sure if we can directly use the entire css from latest web stories. Need to check that out.